src/javascript/EditPanelContainer/EditPanel/EditPanelContent/FormBuilder/SelectorTypes/DateTimePicker/DateTimePicker.jsx

```diff
@@ -29,24 +29,21 @@ export const DateTimePicker = ({id, field, editorContext}) => {
                 // Remove onChange from props pass to the input component as it is set in it.
                 // eslint-disable-next-line react/prop-types
                 const {value, onChange, ...formikField} = props.field;
-                  return (
-                      <DatePickerInput
-                          dayPickerProps={{disabledDays}}
-                          lang={editorContext.uiLang}
-                          initialValue={value ? dayjs(value).toDate() : value}
-                          onChange={
-                            date => {
-                                // TODO: QA-11925 - save date in ISO format without timezone
-                                // eslint-disable-next-line
-                                props.form.setFieldValue(field.formDefinition.name, date, true);
-                            }
-                        }
-                          {...formikField}
-                          displayDateFormat={displayDateFormat}
-                          displayDateMask={displayDateMask}
-                          readOnly={field.formDefinition.readOnly}
-                          variant={variant}
-                          id={id}
+                return (
+                    <DatePickerInput
+                        dayPickerProps={{disabledDays}}
+                        lang={editorContext.uiLang}
+                        initialValue={value ? dayjs(value).toDate() : value}
+                        onChange={date => {
+                            // eslint-disable-next-line
+                            props.form.setFieldValue(field.formDefinition.name, dayjs(date).format('YYYY.MM.DD HH:mm'), true);
```

Comment:
Here we need to get the format from the date format option, `YYYY.MM.DD HH:mm` should be the default value for that option.
This format needs to be use server side as this is how the server  knows how to handle the date format